### PR TITLE
a few initial fixes

### PR DESCRIPTION
**Pull request by atoonk**
https://docs.google.com/spreadsheets/d/1bDv_Oncpuo9-OOJBUXJU1qGlPSCfUhE2rnGdGUpFVBg/edit#gid=0

issue 2 , updated help string for account
issue 3 , removing global persitent flags (yaml config thingy)
issue 9 , removed default long help message
issue 10, change --id to --socket_id for socket delete

### CODE REVIEW

File: go/cmd/socket.go [hunk 1]
```diff
@@ -159,6 +135,6 @@ func init() {
 	socketCreateCmd.Flags().StringVarP(&password, "password", "", "", "Password, required when protected set to true")
 	socketCreateCmd.Flags().StringVarP(&socketType, "type", "t", "http", "Socket type, defaults to http")
 	socketCreateCmd.MarkFlagRequired("name")
-	socketDeleteCmd.Flags().StringVarP(&socketID, "id", "i", "", "Socket ID")
-	socketDeleteCmd.MarkFlagRequired("id")
+	socketDeleteCmd.Flags().StringVarP(&socketID, "socket_id", "i", "", "Socket ID")
```
Comment (btoonk) [inline go/cmd/socket.go]: I think it's better to use the -s as the short flag, now it's -i.

File: go/cmd/account.go [hunk 2]
```diff
@@ -27,6 +27,7 @@ import (
 // accountCmd represents the account command
 var accountCmd = &cobra.Command{
 	Use: "account",
+    Short: "Create a new account or see account information.",
```
Comment (btoonk) [inline go/cmd/account.go]: Go fmt is using tab indenting.

File: go/cmd/socket.go (hunk 1 above)
Comment (atoonk) [reply go/cmd/socket.go]: Agree, I will change it. It's then also consistent with the tunnel cmd which uses -s for socket_id

File: go/cmd/account.go (hunk 2 above)
Comment (atoonk) [reply go/cmd/account.go]: oh nice spotted! replaced space with tab

**Review by btoonk — APPROVED** [review]

**Pull request merged**